### PR TITLE
Add confirm accountId functionality to deauthorize full access key and additional changes for Full Access Keys components

### PR DESCRIPTION
**Pull request by marcinbodnar**
@vgrichina @kcole16 
Question: is Confirm functionality should be available for all full access keys?

I also added changes regarding TODO comments in code, and some additional:
 - fixed publickey display in details box - there was a problem with displaying it for different screen sizes
 - added loading effect for Deauthorize and Confirm buttons, because it takes some time to proceed deauthorize
 - refactor `AccessKeysDeauthorize` component and ContactsRemove since they based on the same CSS from `PaginationSubBox` file.

### CODE REVIEW

File: src/components/access-keys/AccessKeys.js [hunk 1]
```diff
@@ -43,13 +87,16 @@ class AccessKeys extends Component {
...
+
+        this.handleConfirmClear()
     }
 
     handleDeauthorize = () => {
         const publicKey = this.state.showSubData.public_key
 
         this.setState(() => ({
-            loader: true
+            loader: true,
```
Comment (vgrichina) [inline src/components/access-keys/AccessKeys.js:98]: any reason why 2 different loader states are needed? Shouldn't global loader state be enough here?

**Comment by vgrichina** [comment]
@Patrick1904 PTAL, as I have too much stuff in pipeline to do thorough review

File: src/components/contacts/ContactsRemove.js [hunk 2]
```diff
@@ -2,62 +2,71 @@ import React from 'react'
...
+                <b className='color-black'>Alex sent you 1020 Ⓝ</b>
+                <div>3d ago</div>
+            </div>
+            <div className='row border-top'>
+                <b className='color-black'>You and Alex played NEAR Chess</b>
+                <div>1w ago</div>
+            </div>
+        </div>
+        <div className='send'>
+            <Link to='/send-money/marcin'>
```
Comment (Patrick1904) [inline src/components/contacts/ContactsRemove.js:60]: Minor: Probably better to style `Link` component. Button inside of anchor tag is considered [invalid html5](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element).

File: src/components/pagination/PaginationSubBox.js [hunk 3]
```diff
@@ -12,191 +12,250 @@ import CheckBlueImage from '../../images/icon-check-blue.svg'
...
+                    }
+
+                    h2 {
+                        font-size: 32px !important;
+                    }
+                    .title {
+                        padding-bottom: 12px;
+                        .image {
+                            > div {
+                                margin: 0 auto;
```
Comment (Patrick1904) [inline src/components/pagination/PaginationSubBox.js:191]: For future re-factor efforts: Let's utilize styled-components when appropriate to improve readability and make things more modular. Many levels of nesting makes it hard to follow.

File: src/components/access-keys/AccessKeysDeauthorizeConfirm.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    autoCorrect='off'
+                    autoCapitalize='off'
+                    spellCheck='false'
+                    tabIndex='1'
+                    autoFocus={true}
+                />
+            )}
+        </Translate>
+        <div className='alert-info'>
+            {confirmStatus === 'problem' && `Account name doesn't match`}
```
Comment (Patrick1904) [inline src/components/access-keys/AccessKeysDeauthorizeConfirm.js:30]: Minor: Seems string is adequate since there's no variable

File: src/components/pagination/PaginationSubBox.js [hunk 5]
```diff
@@ -12,191 +12,250 @@ import CheckBlueImage from '../../images/icon-check-blue.svg'
...
-
-            :hover {
-               background: #fff;
-               color: #ff585d;
-            }
-         }
-      }
-      .recent-transactions {
-         background-color: #f8f8f8;
-         margin: 0 0 0 0;
```
Comment (Patrick1904) [inline src/components/pagination/PaginationSubBox.js:67]: Minor: `margin: 0` should suffice. There are a few occurrences. Same for `padding` cases.

File: src/components/access-keys/AccessKeys.js (hunk 1 above)
Comment (marcinbodnar) [reply src/components/access-keys/AccessKeys.js:98]: Good point

File: src/components/contacts/ContactsRemove.js (hunk 2 above)
Comment (marcinbodnar) [reply src/components/contacts/ContactsRemove.js:60]: Yes, good point, this part of code is not visible in the project, but the same problem appears in a few other places. My idea was to use <FormButton> component for all form buttons in the project after that linked button was needed and I used it forgetting about that.

The idea with <FormButton> component is good I think, it will be easy to change all buttons if the design will change, also resolves an issue with repeating css.

My proposition is to instead of styling Link, I will prepare component that will be a button with link option (it will be possible to add `to` attr, etc) and we can use it with all buttons. 

I think it's a good idea with having all buttons in one component. What do you think?

File: src/components/pagination/PaginationSubBox.js (hunk 3 above)
Comment (marcinbodnar) [reply src/components/pagination/PaginationSubBox.js:191]: I know what you mean, it's problematic. But also I think that using styled component for small items is problematic to debug too because you end up with hashed classes in dev tools, which makes it harder to find in code.

I think the ideal way to use style components is to make as many reusable components as possible and define more things in GlobalStyles. Personally I don't like when css is defined inside react component because it's a bit messy, that's why in some cases I did put all css in one parent component (for example `SendMoneyContainer.js`), define css for child components there - but of course, it's not a good idea for some cases. 

At this moment css structure is a bit messy because at early stages the design for all pages was changing a lot, many new elements were added. But in the future, it would be best to rebuild the whole css structure, makes more reusable components and have more things in GlobalStyles. Maybe we can also export style components to separate files for every folder or something like that?

Long reply, but I think it's a big issue and we should discuss it in the future.

File: src/components/contacts/ContactsRemove.js (hunk 2 above)
Comment (Patrick1904) [reply src/components/contacts/ContactsRemove.js:60]: Sounds good to me

File: src/components/pagination/PaginationSubBox.js (hunk 3 above)
Comment (vgrichina) [reply src/components/pagination/PaginationSubBox.js:191]: Wonder if we should use this macro to make debugging style-components easier? https://stackoverflow.com/a/60887852/341267

I think I also prefer styles to be encapsulated in components, this seems to scale far better than all big previous projects I worked on which used global styles – they ultimately became very hard to manage mess and with no clear scope which styles impact which component.

So likely best not to use global styles except for basic stuff – like making default styles for HTML tags like `h1`, `h2`, links, etc

Comment (marcinbodnar) [reply src/components/pagination/PaginationSubBox.js:191]: > So likely best not to use global styles except for basic stuff

I agree, what I mean about "have more things in GlobalStyles" was to put in GlobalStyles most of CSS code that is repeated in files and that cannot be put (or it's unnecessary/overdone) to a reusable component. For example all font styles (like you mentioned), border styles, font colors (as it might be mixed for different font styles) etc.

> Not having to think about classNames is a major benefit IMO.

What I meant, showing on examples, is that having separate styled components for Header and Title here:
https://github.com/nearprotocol/near-wallet/blob/master/src/components/accounts/recovery_setup/RecoveryOption.js#L110
seems a bit unnecessary, but I start to think it might be a better approach - JSX structure looks much better for sure.

Comment (vgrichina) [reply src/components/pagination/PaginationSubBox.js:191]: > What I meant, showing on examples, is that having separate styled components for Header and Title here:

agree that it's less clear trade-off in this case. I think biggest advantage of having it as styled component is just that it's easier to extract it into separate reusable one when needed.

File: src/components/access-keys/AccessKeysDeauthorizeConfirm.js (hunk 4 above)
Comment (vgrichina) [reply src/components/access-keys/AccessKeysDeauthorizeConfirm.js]: @marcinbodnar shouldn't this use localized string too?

Comment (marcinbodnar) [reply src/components/access-keys/AccessKeysDeauthorizeConfirm.js]: @vgrichina Good point. Done in #514
